common/src/main/java/tachyon/security/PlainSaslHelper.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the University of California, Berkeley under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package tachyon.security;
+
+import java.security.Security;
+
+import com.google.common.annotations.VisibleForTesting;
+
+public class PlainSaslHelper {
+  // Register Plain SASL server provider
+  static {
+    Security.addProvider(new PlainSaslServerProvider());
+  }
+
+  /**
+   * @param name the name of the provider
+   * @return
+   */
+  @VisibleForTesting
+  public static boolean isPlainSaslProviderAdd() {
```

Comment:
(1) could you update the javadoc?
(2) why we have `@VisibleForTesting` here for a `public static` method? Isn't already visible everywhere?